src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -56,5 +143,24 @@ private void ExportImplicitStyles(bool value)
 			// but, we can add them through merged-dict instead.
 			this.MergedDictionaries.Add(implicitResources);
 		}
+
+		private Style FindStyle(string key)
+		{
+			if (!this.TryGetValue(key, out var resource) || !(resource is Style style))
+			{
+				// uwp: If the {key} style is clearly defined in {info.Source}, but we can't find it here.
+				// And, that it only happens on uwp, and not other uno platforms.
+				// It means that the style references resources that are not directly included.
+				// This can usually be fixed by including `<MaterialColors xmlns="using:Uno.Material" />` in the MergedDictionaries of {info.Source}.
```

Comment:
```suggestion
```
no longer applicable, since they are always included since the addition of xaml-merger